docs/content/envelopes/_index.en.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Secure Envelopes
+date: 2022-03-03T15:36:09-06:00
+lastmod: 2022-03-03T15:36:09-06:00
+description: "Working with Secure Envelopes and cryptography in the TRISA protocol."
+weight: 21
+---
+
+![Secure Envelopes](/img/secure_envelopes.png)
+
+
+The primary data structure for a TRISA exchange is the `SecureEnvelope` -- a wrapper for compliance payload data that facilitates peer-to-peer trust in compliance information exchanges. The design of TRISA secure envelopes had the following requirements:
+
+1. **Privacy**: only the recipient of the secure envelope should be able to open the secure envelope, even outside of the context of an RPC that is secured by transport layer encryption.
+2. **Non-repudiation**: the secure envelope should cryptographically guarantee that the compliance payload is valid and has not been modified or tampered with since the original exchange.
+3. **Long-term data storage**: the secure envelope should be encrypted at rest and can be "erased" by deleting its associated private keys when the compliance statue is over (usually 5-7 years).
+4. **Security**: secure envelopes should prevent statistical attacks on the encrypted payload data while using public-key cryptography for ease of key management.
+
+These requirements are essential to understanding how to successfully engage with TRISA peers, therefore understanding the `SecureEnvelope` is the first step to being able to implement the TRISA protocol.
+
+## Working with Secure Envelopes
+
+There are two basic workflows for secure envelopes: creating and sealing an envelope to send to a counterparty, or unsealing and parsing a received secure envelope.
+
+### Creating a Secure Envelope
+
+**Prerequisites**:
+
+1. You should have constructed an appropriate TRISA `Payload` that contains an `identity` (an IVMS 101 `IdentityPayload`), a `transaction` (a TRISA generic transaction) and a `sent_at` timestamp (RFC-3339 formatted).
+2. You should have the _public sealing key_ of the receipient. You can obtain this key either via the `KeyExchange` RPC or by requesting the key from the directory service.
```

Comment:
I suggest we link the text "directory service" to https://trisa.dev/gds/ (we haven't updated those docs to explain how-to yet, but we will soon!)